### PR TITLE
tests: adapt to new spread version

### PR DESCRIPTION
**Pull request by fgimenez**
Built on top of #1693 , will fail util the new spread binary is made available to travis.

### CODE REVIEW

File: spread.yaml [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
 backends:
     linode:
-        key: $(echo $SPREAD_LINODE_KEY)
+        key: "$(HOST: echo $SPREAD_LINODE_KEY)"
         systems:
             - ubuntu-16.04-64-grub
             - ubuntu-16.04-32-grub
+    qemu:
+        systems:
+            - ubuntu-16.04:
```
Comment (mvo5) [inline spread.yaml]: For consistency, how about we call this `ubuntu-6.04-64-grub` as well? Just like the linode one? This gives us the unity test for free :)

Comment (mvo5) [inline spread.yaml]: Oh, maybe not, lxd is also usuing just `ubuntu-16.04`.

File: spread.yaml [hunk 2]
```diff
@@ -1,21 +1,29 @@
 project: snapd
 
 environment:
-    REUSE_PROJECT: $(echo $REUSE_PROJECT)
+    REUSE_PROJECT: "$(HOST: echo $REUSE_PROJECT)"
     PROJECT_PATH: /home/gopath/src/github.com/snapcore/snapd
     GOPATH: /home/gopath
     PATH: /home/gopath/bin:/snap/bin:$PATH
-    TESTSLIB: $[PROJECT_PATH]/tests/lib
+    TESTSLIB: /home/gopath/src/github.com/snapcore/snapd/tests/lib
```
Comment (mvo5) [inline spread.yaml]: This should work: `$PROJECT_PATH/tests/lib` and avoids the duplication.

File: tests/main/interfaces-cli/task.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ summary: Check the interfaces command
 
 environment:
     SNAP_NAME: network-consumer
-    SNAP_FILE: "./$[SNAP_NAME]_1.0_all.snap"
+    SNAP_FILE: network-consumer_1.0_all.snap
```
Comment (mvo5) [inline tests/main/interfaces-cli/task.yaml]: Hm, I think `SNAP_FILE: $SNAP_NAME_1.0_all.snap` should work.

Comment (mvo5) [inline tests/main/interfaces-cli/task.yaml]: There are a bunch of similar ones that can probably be fixed in a similar way (if you want, it seems not too terrible to just write it explicit here actually).

File: tests/main/refresh-devmode/task.yaml [hunk 4]
```diff
@@ -13,8 +13,8 @@ environment:
     SNAP_NAME: test-snapd-tools
     SNAP_VERSION_PATTERN: \d+\.\d+\+fake1
     CONFIG_DIR: /etc/systemd/system/snapd.service.d
-    STORE_CONFIG: $[CONFIG_DIR]/fakestore.conf
-    BLOB_DIR: $(mktemp -d)
+    STORE_CONFIG: /etc/systemd/system/snapd.service.d/fakestore.conf
```
Comment (mvo5) [inline tests/main/refresh-devmode/task.yaml]: Using `$CONFIG_DIR/...` here might be interessting.

File: spread.yaml (hunk 2 above)
Comment (fgimenez) [inline spread.yaml]: Absolutely, thanks! :) I've taken the opportunity to reuse `$GOPATH` too, please take a look

File: tests/main/interfaces-cli/task.yaml (hunk 3 above)
Comment (fgimenez) [inline tests/main/interfaces-cli/task.yaml]: Sure, done

File: tests/main/refresh-devmode/task.yaml (hunk 4 above)
Comment (fgimenez) [inline tests/main/refresh-devmode/task.yaml]: Thanks, done

**Comment by mvo5** [comment]
Nice, thank you!

**Comment by niemeyer** [comment]
Spread was updated. Tests still not 100%, but better than in master.

**Pull request merged**